pkg/collector/py/utils.go

```diff
@@ -400,3 +402,53 @@ func GetPythonInterpreterMemoryUsage() ([]*PythonStats, error) {
 
 	return myPythonStats, nil
 }
+
+// GetPythonIntegrationList collects python datadog installed integrations list
+func GetPythonIntegrationList() ([]string, error) {
+	glock := newStickyLock()
+	defer glock.unlock()
+
+	pkgModule := python.PyImport_ImportModule(pyPkgModule)
+	if pkgModule == nil {
+		return nil, fmt.Errorf("Unable to import Python module: %s", pyPkgModule)
+	}
+	defer pkgModule.DecRef()
+
+	pkgLister := pkgModule.GetAttrString(pyIntegrationListFunc)
+	if pkgLister == nil {
+		pyErr, err := glock.getPythonError()
+
+		if err != nil {
+			return nil, fmt.Errorf("An error occurred while grabbing the python datadog integration list: %v", err)
+		}
+		return nil, errors.New(pyErr)
+	}
+	defer pkgLister.DecRef()
+
+	args := python.PyTuple_New(0)
+	kwargs := python.PyDict_New()
+	defer args.DecRef()
+	defer kwargs.DecRef()
+
+	packages := pkgLister.Call(args, kwargs)
+	if packages == nil {
+		pyErr, err := glock.getPythonError()
+
+		if err != nil {
+			return nil, fmt.Errorf("An error occurred compiling the list of python integrations: %v", err)
+		}
+		return nil, errors.New(pyErr)
+	}
+	defer packages.DecRef()
+
+	ddPythonPackages := []string{}
+	for i := 0; i < python.PyList_Size(packages); i++ {
+		pkgName := python.PyString_AsString(python.PyList_GetItem(packages, i))
+		if pkgName == "checks_base" || pkgName == "checks-base" {
```

Comment:
nit: I don't think you need to check for the case `checks-base`, if I understand the code in `utils.get_datadog_wheels` correctly we're already replacing `-` with `_`